lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +228,28 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus_manager = corpus_manager
+        self._converter = MystemTagConverter
 
     def _process(self, text: str) -> List[ConlluSentence]:
```

Comment:
this method is also supposed to enrich conllu instances with grammatical info

Reply:
done